source/agora/consensus/EnrollmentManager.d

```diff
@@ -831,6 +766,28 @@ public class EnrollmentManager
         }
     }
 
+    /***************************************************************************
+
+        Update the distance of the pre-image that will be revealed next time
+
+        Params:
+            height = block height to check
+
+    ***************************************************************************/
+
+    public void updateRevealDistance (Height height) @safe nothrow
+    {
+        const enrolled = this.validator_set.getEnrolledHeight(this.enroll_key);
+        if (enrolled == ulong.max)
+            return;
+
+        assert(height >= enrolled);
+        auto dist = cast(uint)(height - enrolled);
+        dist = min(PreimageRevealPeriod * ((dist / PreimageRevealPeriod) + 1),
+                    this.params.ValidatorCycle - 1);
+        this.setNextRevealDistance(dist);
```

Comment:
This is using the same variable for two different values. We should never re-use existing variables for different purposes. Instead I suggest this:

```suggestion
        auto cur_distance = cast(uint)(height - enrolled);
        auto next_distance = min(PreimageRevealPeriod * ((cur_distance / PreimageRevealPeriod) + 1),
                    this.params.ValidatorCycle - 1);
        this.setNextRevealDistance(next_distance);
```

Reply:
> This is using the same variable for two different values. We should never re-use existing variables for different purposes. Instead I suggest this:

How about using `cur` and `next` instead of `cur_distance` and `next_distance` because the modified line would be too long?

Reply:
Sure. Although `curr` is usually the shortened form of `current`.